src/main/views/MattermostBrowserView.test.js

```diff
@@ -17,6 +17,7 @@ import MainWindow from '../windows/mainWindow';
 jest.mock('electron', () => ({
     app: {
         getVersion: () => '5.0.0',
+        getPath: jest.fn(() => '/valid/downloads/path'),
```

Comment:
Not for this PR, but we could consider having a "setup" test file that puts the default in all these functions so we don't have to define (and change it) on every test file.